integration/WIRISplugins.js

```diff
@@ -2261,21 +2261,12 @@ com.wiris.system.JsDOMUtils.addStylesheet = function(d,url) {
 	}
 }
 com.wiris.system.JsDOMUtils.loadTextFile = function(elem,func) {
-	com.wiris.system.JsDOMUtils.loadFileImpl(elem,func);
-}
-com.wiris.system.JsDOMUtils.loadPNGImageFile = function(elem,func) {
-	com.wiris.system.JsDOMUtils.loadFileImpl(elem,func,true,"image/png");
-}
-com.wiris.system.JsDOMUtils.loadFileImpl = function(elem,func,asDataURL,accept) {
-	if(asDataURL == null) asDataURL = false;
```

Comment:
I see a lot of content changes in the JS files that go beyond fixing the encoding. Do you know where this come from?